src/components/SlateEditor/plugins/embed/EditAudio.jsx

```diff
@@ -113,6 +113,7 @@ class EditAudio extends Component {
             />
             <FigureButtons
               figureType="audio"
+              t={t}
```

Comment:
Vits å sende ned t? Og ikke bruker ndla-i18n?

Reply:
Fordi jest feilet for nestede komponenter som tok inn ndla-i18n